core/server/helpers/date.js

```diff
@@ -26,6 +28,13 @@ module.exports = function (date, options) {
     date = date === null ? undefined : date;
 
     format = options.hash.format || 'MMM DD, YYYY';
+
+    // i18n: Making dates, including month names, translatable to any language.
+    // Documentation: http://momentjs.com/docs/#/i18n/
+    // Locales: https://github.com/moment/moment/tree/develop/locale
+    locale = i18n.locale();
+    moment.locale(locale);
```

Comment:
This set's the locale globally, not sure it has unintended side affects in the code base 🤔 I am not sure, therefor we have to go through the code base.

The purpose of this helper is to output dates in the configured locale, maybe we modify the the local moment only for now e.g. `moment().locale` -> only on an instance.

Reply:
I think you're right. Although not side effect currently, that global variable is at least unneeded, and should be local. I will correct it with the next commit.

Reply:
Done.

Reply:
This is still the global moment object.

Reply:
Yes, I made local just that minor `locale` variable used by the `date` helper, while _moment_'s locale is still global.

I've reviewed the many places in Ghost's server and client app where _moment_ is used, and it looks like _moment_'s locale is set by this PR only. So probably no conflict.

On _date.js_, etc., _moment-timezone_ already loads _moment_ (including `moment.locale`) for Node.js.

In my opinion the global _moment_'s locale is what is needed, because creating other instances can cause problems to update the previous ones when changing locale (see _moment_'s issues [1875](https://github.com/moment/moment/issues/1875) and [1797](https://github.com/moment/moment/issues/1797)). Global locale as it is now on this PR is working really well.

Reply:
But if we change the global moment locale that would mean, that the **Ghost server code** operates now in the blog language. That somehow doesn't make sense to me? The default core server code language is still english. Even if this does not conflict, but it feels wrong. 

---

I just noticed that the reading time helper needs an update as well for i18n.
It's using hard coded english texts. See helpers/reading_time.

Reply:
After some testing, now using a local `moment` object to set the locale for dates, with a simple implementation that doesn't seem to cause the problems discussed on the mentioned `moment` issues.

Reply:
As suggested in the review, the `{{reading_time}}` helper is translatable now, keeping backward compatibility with non-translatable themes. Added a small subsection on the reading time helper to the help file.